IdentityCore/src/workplacejoin/MSIDWorkPlaceJoinUtilBase.m

```diff
@@ -114,14 +119,31 @@ + (nullable MSIDWPJKeyPairWithCert *)findWPJRegistrationInfoWithAdditionalPrivat
     queryPrivateKey[(__bridge id)kSecClass] = (__bridge id)kSecClassKey;
     queryPrivateKey[(__bridge id)kSecReturnAttributes] = @YES;
     queryPrivateKey[(__bridge id)kSecReturnRef] = @YES;
+#if !MSID_ENABLE_ECC_SUPPORT
```

Comment:
Any impact on MSAL CPP who might now have this defined? Maybe better to check ifdef as well?

Reply:
I've defined this pre-processor in build settings of Common-core project : https://github.com/AzureAD/microsoft-authentication-library-common-for-objc/blob/e8775297e12225e7bbf9289fc325af38e7e4bb85/IdentityCore/IdentityCore.xcodeproj/project.pbxproj#L9080 
It would be better to add ifdef check as well, I'll add that